nemo/collections/asr/modules/conv_asr.py

```diff
@@ -399,9 +407,14 @@ def forward(self, encoder_output):
         embs = []
 
         for layer in self.emb_layers:
-            pool, emb = layer(pool), layer[:2](pool)
+            pool, emb = layer(pool), layer[: self.emb_id](pool)
             embs.append(emb)
 
+        if self.angular:
+            for W in self.final.parameters():
+                W = F.normalize(W, p=2, dim=1)
```

Comment:
Have you trained a model and checked if you are getting expected output results ?

Reply:
Yes, I was clearing LGTM errors didn't notice. This is corect